src/scenic/syntax/compiler.py

```diff
@@ -43,6 +43,23 @@ def visit_EgoAssign(self, node: s.EgoAssign):
             )
         )
 
+    def visit_Param(self, node: s.Param):
+        d = dict()
+        for parameter in node.elts:
+            d[parameter.identifier] = self.visit(parameter.value)
+        return ast.Expr(
+            value=ast.Call(
+                func=ast.Name(id="param", ctx=loadCtx),
+                args=[
+                    ast.Dict(
+                        [ast.Constant(k) for k in d.keys()],
+                        [v for v in d.values()],
```

Comment:
`d.values()` returns [a dictionary view object](https://docs.python.org/3.7/library/stdtypes.html?highlight=dict%20values#dictionary-view-objects), but the AST takes a list. Here, I'm converting it to a list.

Reply:
It'd be clearer to just write `list(d.values())` I think -- unless you're trying to mirror the syntax on the previous line?

Reply:
Fixed in 41573b2572e0cf7246e1fc45ad57819c57ea5b3d